spec/factories.rb

```diff
@@ -16,4 +16,17 @@
     state 'MA'
     zip_code '02113'
   end
+
+  factory :review do
+    sequence(:title) { |n| "#{n} This is my pizza slice" }
+    sequence(:body) { |n| "#{n} Literally stumbled upon this place. I had been hearing 
```

Comment:
Line is too long. [87/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.